tests/cases.py

```diff
@@ -197,6 +197,7 @@ def assert_all_data_types_row(
     timestamp_precision: int = 6,
     schema: TTableSchemaColumns = None,
     expect_filtered_null_columns=False,
+    allow_string_binary: bool = False,
```

Comment:
on clickhouse,  loading a binary row from parquet end's up being a base64 string and loading from jsonl ends up being a binary just converted to a string (or vice versa, forgot, you can see it in the tests). I will mention this in the docs, I supposed we'll have to live with it.